ingenialink/servo.py

```diff
@@ -41,6 +47,87 @@
 OPERATION_TIME_OUT = -3
 
 
+class DictionaryFactory:
+    """Dictionary factory, creates a dictionary instance depends on the file version
```

Comment:
Based on the dictionary version and communication type.

Reply:
Dictionary factory, creates the appropriate dictionary instance according to the file version and connection interface